docs/navigation.md

```diff
@@ -0,0 +1,114 @@
+# Navigation panel
+
+This guide will walk you through the functions of each button in navigation panel. 
+
+# Navigation Button
+
+The first button in the navigation panel is the Navigation button. It is one of the default active buttons.
+
+Navigation button is divided into four parts. It provides the following information:
+
+- Servers
+- Operations
+- Messages
+- Schemas
+
+## Servers
+
+The Servers section displays the name and information of the [server]( https://www.asyncapi.com/docs/concepts/server) you are using to build your AsyncAPI.
+
+## Operations
+
+The Operations section lists all the operations that are a part of the AsyncAPI. It adds the perfix `PUB`  for publisher and `SUB` for subscriber to differentiate between both the operations.
+
+## Messages
+
+This Messages section displays the messages stored in `components.messages` section. This does not mean that every message in the component is used.  It can be defined but it may not used by references in the `operation` field.
```

Comment:
```suggestion
The Messages section displays the messages stored in the `components.messages` section. This does not mean that every message in the component is used.  It can be defined but it may not used by references in the `operation` field.
```